### PR TITLE
New feature: Copy Command line arguments to clipboard.

### PR DESCRIPTION
**Pull request by mibcoder**
Hello MBulli

I Added a new command (toolbar button) that allows to copy the currently selected command line arguments as a string to the clipboard. This is useful i.e. for testing a set of arguments outside of visual studio in the console window or for adding them to documentation as an example.

### CODE REVIEW

**Comment by MBulli** [comment]
Thanks for the PR. That's a nice feature.

We definitely going to merge it but there are some things I would like to have changed or discussed.
First I have to ask you to fix the URLs in the readme so that they point to this repo again.

The second thing is about ux. We already have the usual copy/paste functionality bound to ctrl+c/v.
The icon you choose for your new feature suggest that it invokes the same copy/paste logic and implies that the copy button is missing. Thus I'm suggesting to use a different icon. What do you think?

Also about ux, the user has no chance to know what happened after the toolbar button was clicked. You have to know that you now have something in the clipboard. My suggestion is to show a minimalistic window with a single textbox showing the commandline and a button which invokes the clipboard code.

**Comment by mibcoder** [comment]
Hello MBulli

Thank you for considering...
I would acknowledge 1+2, however #3 seems to me a little bit too much. I don't know of any application, which requires confirmation before placing data in the clipboard or provide feedback. The user should understand the functionality by reading the tool-tip and recognizing the (then modified) "copy to clipboard" symbol.

**Comment by MBulli** [comment]
I'm fine with that. Would you mind to fix the URLs in the Readme?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,10 @@
 # Smart Commandline Arguments 
 A Visual Studio Extension which aims to provide a better UI to manage your command line arguments
 
+This fork added a new command that allows to copy the currently selected command line arguments as a string to the clipboard. This is useful i.e. for testing a set of arguments outside of visual studio in the console window or for adding them to documentation as an example.
```
Comment (MBulli) [inline README.md]: Please remove that line

File: README.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+
 ## Install
-Install the extension inside Visual Studio or download it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=MBulli.SmartCommandlineArguments "Visual Studio Marketplace").
+Install the extension inside Visual Studio or download it from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=mibcoder.SmartCommandlineArguments "Visual Studio Marketplace").
```
Comment (MBulli) [inline README.md]: Replace mibcoder with mbulli

File: README.md [hunk 3]
```diff
@@ -11,7 +13,7 @@ Visual Studio 2015 and 2017 and the following project types are supported:
 - Node.js
 - Python (requires a nightly build of PTVS)
 
-If you're using Cmake make sure to read the [Cmake support wiki page](https://github.com/MBulli/SmartCommandlineArgs/wiki/Cmake-support "Cmake").
+If you're using Cmake make sure to read the [Cmake support wiki page](https://github.com/mibcoder/SmartCommandlineArgs/wiki/Cmake-support "Cmake").
```
Comment (MBulli) [inline README.md]: Replace mibcoder with mbulli

File: README.md [hunk 4]
```diff
@@ -26,11 +28,12 @@ If this new behavior is not welcomed one can fallback to the 'old' mode where th
 Tools → Options → Smart Commandline Arguments → General → Enable version control support
 
 ## Interface
-![alt text](https://github.com/MBulli/SmartCommandlineArgs/blob/master/Doc/Images/AddIcon.png "Add Button"): Add new line  
-![alt text](https://github.com/MBulli/SmartCommandlineArgs/blob/master/Doc/Images/RemoveIcon.png "Remove Button"): Remove selected lines  
-![alt text](https://github.com/MBulli/SmartCommandlineArgs/blob/master/Doc/Images/MoveUpIcon.png "Move Up Button") / ![alt text](https://github.com/MBulli/SmartCommandlineArgs/blob/master/Doc/Images/MoveDownIcon.png "Move Down Button"): Move selected lines  
-  
-![alt text](https://github.com/MBulli/SmartCommandlineArgs/blob/master/Doc/Images/example.png "Commandline Arguments Window")
+![alt text](https://github.com/mibcoder/SmartCommandlineArgs/blob/master/Doc/Images/AddIcon.png "Add Button"): Add new line
```
Comment (MBulli) [inline README.md]: Replace mibcoder with mbulli in all following lines

**Comment by mibcoder** [comment]
Done - and thanks for reviewing.

**Comment by MBulli** [comment]
Thank you!

**Comment by mibcoder** [comment]
Yeah, my first contribution..... :-)

**Comment by MBulli** [comment]
Congratulations! Don't let it be the last one ;)

**Pull request merged**